### PR TITLE
Addresses #12 by adding in a panel with all the Github issues for the…

### PR DESCRIPTION
**Pull request by sskhandek**
… site

@kevinwang This works as per the attached screenshot, but I feel like there's a better way to handle refreshing the data than just setting a timeout that goes off every 12 hours. Thoughts?

screen shot 2016-12-15 at 12 06 02 am

### CODE REVIEW

File: src/panels/githubissues.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    },
+
+    updateData: function() {
+        $.getJSON('https://api.github.com/repos/IllinoisWCS/IllinoisWCS.github.io/issues')
+            .then((data) => {
+                this.setState({
+                    issues: data
+                })
+            });
+        setTimeout(this.updateData, 12 * 60 * 60 * 1000)
```
Comment (kevinwang) [inline src/panels/githubissues.js]: Missing semicolon. Additionally, I recommend using `setInterval` to match how the other panels handle polling ([example](https://github.com/IllinoisWCS/wcs-display/blob/master/src/panels/mtd.js#L20)).

File: src/panels/githubissues.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ */
+var IssuesPanel = React.createClass({
+    getInitialState: function() {
+        return {
+            issues: []
+        };
+    },
+
+    componentDidMount: function() {
+        $.getJSON('https://api.github.com/repos/IllinoisWCS/IllinoisWCS.github.io/issues')
```
Comment (kevinwang) [inline src/panels/githubissues.js]: Why is this request duplicated here and in `updateData`? That means the request would be fired twice on mount.

**Review by kevinwang — CHANGES_REQUESTED** [review]

**Comment by sskhandek** [comment]
@kevinwang made updates. Could you re-check? Thanks!

**Comment by kevinwang** [comment]
It appears that you have just removed the API call altogether?

**Comment by sskhandek** [comment]
@kevinwang I'm an idiot. I refactored and I thought I checked, but I didn't run npm build. Putting it back

**Comment by sskhandek** [comment]
@kevinwang Let's try that again. I put the API call back and I ran the build and it works

**Review by kevinwang — APPROVED** [review]

**Pull request merged**